I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/market_minigame.rpy", line 162, in script
    jump d2_after_work2
ScriptError: could not find label 'd2_after_work2'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/market_minigame.rpy", line 162, in script
    jump d2_after_work2
  File "renpy/ast.py", line 1974, in execute
    rv = renpy.game.script.lookup(target)
  File "renpy/script.py", line 927, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'd2_after_work2'.

Windows-10-10.0.19041 AMD64
Ren'Py 7.5.2.22081402
RPG Battle System Test 1.0
Sat Sep  9 16:34:47 2023
